File: Day 94/classwork/classwork.py
arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0]

arr2 = list(filter(lambda x : x % 12 == 0, arr))


def count_red_beads(n):
    if(n < 2):
        return 0

    return (n - 1) * 2

def small_enough(array, limit):
    for i in array:
        if(limit < i and i != limit):
            return False
    return True

File: Day 135/classwork/classwork.js
function func(arr){
    let arr2 = []
    for(let i=0; i<arr.length; i++){
        for(let k in arr){
            if(i != k && arr[i] > arr[k]){
                arr2.push(k * Math.abs(i - k - 1))
            }
            else if(i != k && arr[i] < arr[k]){
                arr2.push(i * Math.abs(i - k - 1))
            }
        }
    }
    return arr2
    
}
console.log(func([1,8,6,2,5,4,8,3,7]))